pulsar-functions/runtime/pom.xml

```diff
@@ -50,6 +50,13 @@
       <artifactId>jcommander</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
Can you define `snakeyaml` in the root pom file? So that you can just reference the artifact without specifying the version. Pulsar manages all the versions in the root pom file.

Reply:
@wolfstudy can we not use Jackson which is already dependency to write the yaml file?